### DISCUSSION TITLE
Lodestar Standup 2024-40 (October 15, 2024)

### DISCUSSION THREAD

**Discussion by philknows** [ChainSafe/lodestar#7136] [Standups]
### Meeting Info
[October 15, 2024, 14:00-15:00 UTC](https://savvytime.com/converter/utc-to-germany-berlin-united-kingdom-london-ny-new-york-city-ca-san-francisco-thailand-bangkok-japan-tokyo-australia-sydney/oct-15-2024/2pm)
Zoom: shared on #lodestar-general ChainSafe Discord channel at 14:00 UTC

### Agenda
- v1.23.0 Metrics Review + RC debugs/additions
- Open Discussion
- Team Updates
- Closing Remarks

### Instructions
Please post here any topics to be added for Standup discussions. Examples of agenda items are issues, discussions, proposals which require feedback, consensus from the team and/or community members.

**Comment by philknows** [comment]
Would like to gauge consensus on pushing our standup to 15:00 UTC post-Devcon when daylight savings ends for North America and Europe.

Current Standup:
![Screenshot 2024-10-10 at 10 51 26 AM](https://github.com/user-attachments/assets/5b3146a1-8fc8-41cf-b9fc-722e218d41b9)

Proposed Standup:
![Screenshot 2024-10-10 at 10 52 11 AM](https://github.com/user-attachments/assets/821480a5-3540-41cd-9cb2-48518ec8e6e8)

**Reply by ensi321** [reply]
That's pretty good. Or else it will be 6am in the winter for me and I will be giving updates as if I am sleep talking.

**Comment by twoeths** [comment]
we need to release ssz https://github.com/ChainSafe/ssz/pull/399
this is required for EIP-4881 at lodestar side which is supposed to be v1.24, StableContainer is ready there too

**Comment by nflaig** [comment]
would be good to quickly discuss how we feel about including https://github.com/ethereum/consensus-specs/pull/3900 in Pectra

**Comment by philknows** [comment]
Closed for #7166
